aten/src/ATen/native/Activation.cpp

```diff
@@ -25,6 +25,16 @@ Tensor & selu_(Tensor & self) {
   return at::elu_(self, SELU_ALPHA, SELU_SCALE);
 }
 
+Tensor celu(const Tensor & self, Scalar alpha) {
+  Tensor inv_alpha = alpha.toTensor().reciprocal();
```

Comment:
Can you instead do `double inv_alpha = 1. / alpha.to<double>()`? This avoids launching functions optimized for large tensors.